### ISSUE TITLE
azure backend configuration change

### ISSUE THREAD

**Issue by jasminpatel99** [charleszipp/azure-pipelines-tasks-terraform#134]
I see following errors in app version 0.5.9  but don't see these errors in 0.5.2

2020-05-15T12:34:11.0153718Z ==============================================================================
2020-05-15T12:34:11.0154113Z Task         : Terraform CLI
2020-05-15T12:34:11.0154430Z Description  : Execute terraform cli commands
2020-05-15T12:34:11.0154732Z Version      : 0.5.9
2020-05-15T12:34:11.0154987Z Author       : Charles Zipp
2020-05-15T12:34:11.0155281Z Help         : 
2020-05-15T12:34:11.0155588Z ==============================================================================
2020-05-15T12:34:11.4483254Z [command]/opt/hostedtoolcache/terraform/0.12.25/x64/terraform version
2020-05-15T12:34:11.4857042Z Terraform v0.12.25

2020-05-15T12:34:11.5425141Z [1m[33mWarning: [0m[0m[1m"arm_client_secret": [DEPRECATED] `arm_client_secret` has been replaced by `client_secret`[0m
2020-05-15T12:34:11.5425662Z 
2020-05-15T12:34:11.5426251Z [0m[0m[0m
2020-05-15T12:34:11.5426842Z [33m
2020-05-15T12:34:11.5429576Z [1m[33mWarning: [0m[0m[1m"arm_subscription_id": [DEPRECATED] `arm_subscription_id` has been replaced by `subscription_id`[0m
2020-05-15T12:34:11.5430210Z 
2020-05-15T12:34:11.5430740Z [0m[0m[0m
2020-05-15T12:34:11.5431421Z [33m
2020-05-15T12:34:11.5497546Z [1m[33mWarning: [0m[0m[1m"arm_client_id": [DEPRECATED] `arm_client_id` has been replaced by `client_id`[0m
2020-05-15T12:34:11.5497864Z 
2020-05-15T12:34:11.5545363Z [0m[0m[0m
2020-05-15T12:34:11.5546358Z [33m
2020-05-15T12:34:11.5548071Z [1m[33mWarning: [0m[0m[1m"arm_tenant_id": [DEPRECATED] `arm_tenant_id` has been replaced by `tenant_id`[0m
2020-05-15T12:34:11.5550745Z 
2020-05-15T12:34:11.5551354Z [0m[0m[0m
2020-05-15T12:34:11.5551938Z [31m
2020-05-15T12:34:11.5553195Z [1m[31mError: [0m[0m[1mError building ARM Config: Azure CLI Authorization Profile was not found. Please ensure the Azure CLI is installed and then log-in with `az login`.[0m
2020-05-15T12:34:11.5554299Z 
2020-05-15T12:34:11.5554900Z [0m[0m[0m
2020-05-15T12:34:11.5555173Z 
2020-05-15T12:34:11.5589977Z ##[error]Terraform command 'init' failed with exit code '1'.:  Error building ARM Config: Azure CLI Authorization Profile was not found. Please ensure the Azure CLI is installed and then log-in with `az login`.
2020-05-15T12:34:12.3139323Z ##[section]Finishing: terraform init

**Comment by charleszipp** [comment]
@jasminpatel99 would it be possible for you to share the template that is producing this error as well as how you have the TerraformCLI task configured for init?

**Comment by charleszipp** [comment]
@jasminpatel99 I suspect that the service connection you are attempting to use and the underlying service principal credentials are invalid. I believe if you click the refresh button to the right of the Backend Azure Subscription drop down, it should give you an error if the connection you are trying to use is not authorized for the subscription. Additionally, your user may not have access to use a service connection or, create one if one does not already exist.

**Comment by jasminpatel99** [comment]
I am the owner of the subscription but things could go wrong. I could manually spin up tf files ok. When I refreshed the button to the right of the Backend Azure Subscription drop down, it shows following message. (I created SP for this, i also tried picking my subscription directly and authorizing it)
"Scoped to resource group 'AksTerraform-RG'"

AksTerraform-RG is my resource group for terraform backend.

Thanks for the prompt response.
